examples/examplesPriceAggregator/main.go

```diff
@@ -120,6 +124,17 @@ func runApp() error {
 	return nil
 }
 
+func addPairsToFetchers(pairs []*aggregator.ArgsPair, priceFetchers []aggregator.PriceFetcher) {
+	for _, pair := range pairs {
+		for _, fetcher := range priceFetchers {
```

Comment:
Please split this last for in a new function. Might add also a log print if the fetcher name is not registered on the Exchanges map